fhir/immunizations_demo/docs/README.md

```diff
@@ -33,9 +33,25 @@ a project and configure the Cloud Healtcare API. However, instead of creating a
 service account, follow the instructions on the [OAuth
 quickstart](https://cloud.google.com/docs/authentication/end-user). You
 will only need the client ID, the client secret (API key) is not required for
-this use case. You will also need to register
+this use case.
+
+You will also need to register
 `http://localhost:4200/oauth-callback` as an authorized redirect URI on the
-OAuth configuration page. Change `localhost:4200` if you plan to develop on a
+OAuth configuration page. Follow these steps:
+
+1. If you haven't already, select **APIs & Services** from the left-hand menu of the Google Cloud Platform dashboard.
```

Comment:
I wouldn't assume that someone has already started poking around. "Go to the **APIs & Services** page using the left-hand menu of the Google Cloud Platform dashboard."

Reply:
Roger that :) Thanks for the feedback!